pkg/volume/iscsi/iscsi_util.go

```diff
@@ -300,7 +300,7 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) (string, error) {
 	// mount it
 	globalPDPath := b.manager.MakeGlobalPDName(*b.iscsiDisk)
 	notMnt, err := b.mounter.IsLikelyNotMountPoint(globalPDPath)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
Should add a comment here to show that the directory is created below and that is why os.IsNotExist() is avoided. Or, reorganize the flow.